Color/src/Graphics/Color/Algebra/Binary.hs

```diff
@@ -134,6 +134,8 @@ instance Elevator Bit where
   toShowS _       = ('1':)
   toWord8 = coerce
   {-# INLINE toWord8 #-}
+  fromWord8 = coerce
```

Comment:
This isn't quite accurate. Bit has only two values `zero` and `one`, which have their own underlying representation `0x00` and `0xff` respectively. Coercion would violate this invariant.
```suggestion
  fromWord8 0 = zero
  fromWord8 _ = one
```